blueman/bluez/Base.py

```diff
@@ -131,6 +142,10 @@ def get_properties(self) -> Dict[str, Any]:
 
         return props
 
+    def destroy(self) -> None:
```

Comment:
It's "on purpose" that this does not get used (yet), right?

Reply:
It's used here :-) https://github.com/blueman-project/blueman/blob/ac6d54668c4d5d13e510c5f4cc28f7dd643af25b/blueman/gui/manager/ManagerDeviceList.py#L134

Reply:
Ah, sure, this is an override. :facepalm: